### PR TITLE
UCP/WIREUP: Add am_bw lanes - for multi-rail active messages.

### PR DESCRIPTION
**Pull request by yosefe**

### CODE REVIEW

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/5790/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/5798/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/5800/ for details (Mellanox internal link).

File: src/ucp/core/ucp_ep.c [hunk 1]
```diff
@@ -1153,6 +1165,9 @@ static void ucp_ep_config_print(FILE *stream, ucp_worker_h worker,
          }
      }
 
+     if (context->config.features & UCP_FEATURE_TAG) {
+     }
```
Comment (hoopoepg) [inline src/ucp/core/ucp_ep.c]: empty statement?

File: src/ucp/wireup/wireup.c [hunk 2]
```diff
@@ -510,16 +516,31 @@ static ucs_status_t ucp_wireup_resolve_proxy_lanes(ucp_ep_h ep)
         }
 
         /* Create a signaling ep to the proxy lane */
-        status = ucp_signaling_ep_create(ep, ep->uct_eps[proxy_lane],
-                                         proxy_lane == lane, &signaling_ep);
-        if (status != UCS_OK) {
-            /* coverity[leaked_storage] */
-            return status;
+        if (proxy_lane == lane) {
+            /* If proxy is to the same lane, temporarily remote the existing
```
Comment (hoopoepg) [inline src/ucp/wireup/wireup.c]: s/remote/remove/ ???

**Comment by hoopoepg** [comment]
it seems tests are failed due to processing error after EP is destroyed (double error processing)

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/5808/ for details (Mellanox internal link).

**Comment by yosefe** [comment]
@hoopoepg pls take a look (tests failed on coverity, i'll fix that soon)

**Comment by hoopoepg** [comment]
@yosefe it is fine for me

**Comment by hoopoepg** [comment]
@yosefe I looked at error - it seems coverity reports false positive, may be it can't process ```({statement})``` construction

**Comment by yosefe** [comment]
@hoopoepg indeed, last commit should help

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/5817/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/5818/ for details (Mellanox internal link).

**Pull request merged**